### DISCUSSION TITLE
Applied for student developer pack, says approved but `PRO` doesn't show on profile, and message says `verified as null`

### DISCUSSION THREAD

**Discussion by yatindra75** [community/community#114338] [GitHub Education]
### Hi everyone,

Hello folks,

I finished my verification for the developer pack.

![Screenshot 2024-03-26 at 09-06-29 Discounted and free plans are available for educational use](https://github.com/community/community/assets/98309237/08147e88-85ce-4028-84cb-25097c7ead87)

But 
1. Profile doesn't show the `PRO` badge
2. I did not receive a mail for the acceptance
3. The following message shows on the application page- 

![image](https://github.com/community/community/assets/98309237/5a1ba44f-e778-4114-abf7-cb69fa5cebbd)

I wish to confirm if this is acceptable, and if there is some action to be taken from my side, and lastly, when should I expect to be able to access the benefits?

Important note- Right before the accepted application, one application got rejected as I hadn't set the billing info.

**Comment by namith1003** [comment]
Give it a few more days, even though it says approved it is very misleading but as per the instructions it's not complete yet and requires a few more days give it up to a week and it should change like this. It took me about 5 days, some people less and some people more.

![Screenshot 2024-03-26 134448](https://github.com/community/community/assets/65733592/884a4c66-965f-4e66-9ab2-4ba304203367)


Hopefully, this answers your question.

- Namith

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
